resources/jenkins/README.md

```diff
@@ -0,0 +1,3 @@
+These scripts are intended to be executed within a Jenkins build
```

Comment:
I think these are superfluous. I can't of anything other than `./build.sh` we'd want to add to the jenkins-specific script.

Reply:
@bbaldino we didn't go out of our way to remove them in rtp and jmt, but I think you'll agree we don't need to add them

Reply:
I just copied what was in jitsi-rtp -- I don't know what's actually needed.

Reply:
Yeah we can skip them

Reply:
So just `resources/build.sh` and not anything in `resources/jenkins`?

Reply:
Yes

Reply:
Yeah just make sure build.sh doesn't reference Jenkins vars